src/main/kotlin/racingcar/RandomUtil.kt

```diff
@@ -0,0 +1,5 @@
+package racingcar
+
+import kotlin.random.Random
+
+fun getZeroToRange(range: Int) = Random.nextInt(range + 1)
```

Comment:
```suggestion
fun getZeroToRange(range: Int) = 0..9.random()
```

이렇게 하면 더 깔끔하지 않을까요~?

Reply:
이런게 있었군요! 감사합니다